msal-dotnet-articles/how-to/token-cache-serialization.md

```diff
@@ -416,9 +375,7 @@ cacheHelper.RegisterCache(pca.UserTokenCache);
 
 #### Plain-text fallback mode
 
-The cross-platform token cache allows you to store unencrypted tokens in plain text. This feature is intended for use in development environments for debugging purposes only. 
-
-You can use the plain-text fallback mode by using the following code pattern.
+The cross-platform token cache allows you to store unencrypted tokens in an ACL-restricted plain-text file. This is useful in cases where encryption at rest fails, which ocassionally happens due to environmnent-related reasons. You can use the plain-text fallback mode by using the following code pattern:
```

Comment:
Update this text above:

> In desktop applications, we recommend that you use the cross-platform token cache. MSAL.NET provides the cross-platform token cache in a separate library named [Microsoft.Identity.Client.Extensions.MSAL](https://github.com/AzureAD/microsoft-authentication-extensions-for-dotnet).

to this:
> In desktop applications, we recommend that you use the cross-platform token cache. MSAL.NET provides the cross-platform token cache in a separate library named [Microsoft.Identity.Client.Extensions.MSAL](https://www.nuget.org/packages/Microsoft.Identity.Client.Extensions.Msal).

Also update the references to the archives cache extensions repo to main MSAL repo.

Reply:
Added a separate issue for this: #363